### PR TITLE
Enhancement/noid/hash service

### PR DESCRIPTION
**Pull request by georgehrke**
implements a hash_service

needed for:
https://github.com/nextcloud/calendar/issues/21
https://github.com/nextcloud/calendar/issues/41

### CODE REVIEW

**Comment by georgehrke** [comment]
fixed the missing semicolon

File: js/Gruntfile.js [hunk 1]
```diff
@@ -136,6 +136,7 @@ module.exports = function(grunt) {
 		babel: {
 			options: {
 				presets: ['es2015'],
+				plugins: ["array-includes"],
```
Comment (tcitworld) [inline js/Gruntfile.js]: Hmm, seems like `String.prototype.includes` (and same with array) is already available for our requirements : https://kangax.github.io/compat-table/es6/#test-String.prototype.contains

Comment (georgehrke) [inline js/Gruntfile.js]: Good catch!
Will remove it.

**Comment by georgehrke** [comment]
removed the babel plugin, let's wait for the tests

**Pull request merged**